Northstar.Client/mod/scripts/vscripts/_northstar_client_init.nut

```diff
@@ -0,0 +1,22 @@
+global struct GameStateStruct {
```

Comment:
bit of a nitpick, but all vars in this struct have names whose formatting aren't consistent with scripts as a whole, or the rest of the code in this pr